config/toml.go

```diff
@@ -284,6 +284,11 @@ external_address = "{{ .P2P.ExternalAddress }}"
 # Comma separated list of seed nodes to connect to
 seeds = "{{ .P2P.Seeds }}"
 
+# Comma separated list of peers to be added to the peer store
+# on startup. Either BootstrapPeers or PersistentPeers are
```

Comment:
```suggestion
# on startup. Either bootstrap_peers or persistent_peers are
```

The configuration file has no notion of `BootstrapPeers` - this is the Go code field name. The representation in the config file is `bootstrap_peers`.